src/libraries/System.Drawing.Common/src/System/Drawing/GdiplusNative.cs

```diff
@@ -1374,8 +1374,11 @@ internal struct StartupInput
 
             public static StartupInput GetDefault()
             {
+                OperatingSystem os = Environment.OSVersion;
                 StartupInput result = default;
-                result.GdiplusVersion = 1;
+
+                // In Windows 7 GDI+1.1 story is different as there are different binaries per GDI+ version.
+                result.GdiplusVersion = os.Platform == PlatformID.Win32NT && os.Version.Major == 6 && os.Version.Minor == 1 ? 1 : 2;
```

Comment:
```suggestion
                bool isWindows7 = (os.Platform == PlatformID.Win32NT && os.Version.Major == 6 && os.Version.Minor == 1);
                result.GdiplusVersion =  isWindows7 ? 1 : 2;
```
might be clearer -- since I was immediately thinking "what about 6.0 and 6.2"

Reply:
Yeah makes sense.